wordpress-mobile/WordPress-iOS
# Stop using attachments to post has possible featured images.

Fixes #3612

Sometimes images are attached and then removed from the content, so they should'n be used as possible feature images.

To test:
 - Launch the app
 - Create a new post
 - Tap on add media and select one new image
 - After the image is uploaded remove it from the post content
 - Save or Publish the post
 - Check that in the post list the image don't shows up

Needs review: @aerych
